### ISSUE TITLE
Creating Asset

### ISSUE THREAD

**Issue by ThamarD** [GNYIO/gny-wallet#62]
While creating an asset, in the description field I filled in the following description "'@#$ - Thamar - '@#$", or something like that. I think that did kill the chain.

Sorry :-)

Found the following in the logs:

```
db1      | STATEMENT:  insert into "asset" ("tid", "name", "timestamp", "desc", "maximum", "precision", "quantity", "issuerId", "_version_") values ('b73aad548a70d6c7724452059c0c594036989ae05be2ff12815c96e1fee42b49', 'thamar.THMR', 67715841, '&!@#' - Thamar tést coin - &!@#'', '50000000000000000', 8, '0', 'G2oyuwFiRQvd3QvXQtxMnYf4br5rb', 1);
```

This should give you enough info.

**Comment by a1300** [comment]
This bug can be fixed with the RegExp `/^([A-Za-z]+ )+[A-Za-z]+$|^[A-Za-z]+$/`
> This RegExp **must** be used with a check for `null` or `undefined`

Unfortunately `RegExp` is casting all input to `strings`. Therefore in a `node.js` REPL:
```
> /^([A-Za-z]+ )+[A-Za-z]+$|^[A-Za-z]+$/.test(undefined);
> true // because undefined gets casted as a string
> /^([A-Za-z]+ )+[A-Za-z]+$|^[A-Za-z]+$/.test(null);
> true // because  null gets casted to a string
```

**Comment by a1300** [comment]
Closed with PR: #63 

Thanks @ThamarD for reporting!
